atomicapp/providers/openshift.py

```diff
@@ -313,6 +365,9 @@ def _parse_kubeconf(self, filename):
             if co["name"] == current_context:
                 context = co
 
+        if not context:
+            raise ProviderFailedException()
```

Comment:
without this check, the subsequent code used to break when ``context == None``, as they check for ``context[somekey]``.